acoustid/data/submission.py

```
@@ -67,6 +67,24 @@ def import_submission(ingest_db, app_db, fingerprint_db, index_pool, submission)
     Import the given submission into the main fingerprint database
     """
 
+    if not pg_try_advisory_xact_lock(
+        ingest_db, "import.fp", str(submission["fingerprint"])
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)